modules/games.js

```diff
@@ -0,0 +1,318 @@
+        let faction = "No Faction";
+        const distance = Math.round(station.distanceToArrival * 10) / 10;
+        if (station.controllingFaction) {
+          faction = station.controllingFaction.name;
+        }
+        embed.addField(faction, "[" + station.name + "](" + encodeURI(stationURL) + ")\n" + distance + " ls", true);
+      }
+    }
+
+    // Letting the user know there were more than 25
+    if (stationList.size > 25) embed.setFooter({ text: "Some stations were filtered out because the limit was exceeded.", iconURL: "https://i.imgur.com/vYPj8iX.png" });
+    inter.reply({ embeds: [embed] });
+  };
+  const getFactions = () => {
+    if (!starSystem) return inter.reply({ content: "I couldn't find a system with that name.", ephemeral: true });
+    if (starSystem.factions.length < 1) return inter.reply({ content: "I couldn't find any factions in that system.", ephemeral: true });
+    embed.setTitle(starSystem.name).setURL(starSystem.factionsURL);
+
+    for (const faction of starSystem.factions) {
+      const influence = Math.round(faction.influence * 10000) / 100;
+      const url = encodeURI(`https://www.edsm.net/en/faction/id/${faction.id}/name/`);
+      embed.addField(faction.name + (starSystem.information && (faction.name === starSystem.information.faction) ? " (Controlling)" : "") + " " + influence + "%",
+        "State: " + faction.state + "\nGovernment: " + faction.allegiance + " - " + faction.government + "\n[Link](" + url + ")", true);
+    }
+    return inter.reply({ embeds: [embed] });
+  };
+  const getBodies = () => {
+    if (!starSystem) return inter.reply({ content: "I couldn't find a system with that name.", ephemeral: true });
+    if (starSystem.bodies.length < 1) return inter.reply({ content: "I couldn't find any bodies in that system.", ephemeral: true });
+    embed.setTitle(starSystem.name).setURL(starSystem.bodiesURL);
+
+    for (const body of starSystem.bodies) {
+      const scoopable = body.type === "Star" ? (body.isScoopable ? " (Scoopable)" : " (Not Scoopable)") : "";
+      const distance = Math.round(body.distanceToArrival * 10) / 10;
+      embed.addField(body.name, body.type + scoopable + "\n" + distance + " ls", true);
+    }
+    return inter.reply({ embeds: [embed] });
+  };
+  const info = inter.options.getString('info');
+  switch (info) {
+  case "status": return getStatus();
+  case "time": return getTime();
+  case "bodies": return getBodies();
+  case "factions": return getFactions();
+  case "stations": return getStations();
+  case "system": return getSystem();
+  }
+}
+
+/** @param {discord.CommandInteraction} inter */
+async function minecraftSkin(inter) {
+  const user = inter.options.getMember('user');
+  let name = inter.options.getString('username');
+  if (user) name ??= await Module.db.ign.find(user.id ?? inter.user.id, 'minecraft')?.ign;
```

Comment:
This fallback does not work as intended, because it only triggers if `user` is provided.